src/App.vue

```diff
@@ -190,6 +190,9 @@ export default {
         console.log("sending this message to CATAN SERVER: " + joinMessage);
         this.socket.send(joinMessage);
         this.hide_join_modal();
+    },
+    createAction: function (action,object,resources) {
+      this.actions.push({action: action, object: object, resources: resources});
```

Comment:
object kan misschien beter key heten? of objectkey?

Reply:
Ik heb uiteindelijk het hele object doorgegeven, dat was makkelijk.